### PR TITLE
Add GH test action

### PR DESCRIPTION
**Pull request by Zejnilovic**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Run Tests
+
+on:
+  push:
+    branches: [ maste, develop ]
```
Comment (dk1844) [inline .github/workflows/test.yml]: master?

**Review by dk1844 — CHANGES_REQUESTED** [review]
Otherwise LGTM.

**Review by AdrianOlosutean — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (dk1844) [reply .github/workflows/test.yml]: I can see you fixed it already

Comment (Zejnilovic) [reply .github/workflows/test.yml]: fixed in the commit before the comment

**Review by dk1844 — APPROVED** [review]

**Pull request merged**